clusterloader2/testing/experimental/storage/pod-startup/persistent-volumes/config.yaml

```diff
@@ -0,0 +1,142 @@
+# ASSUMPTIONS:
+# TODO: Assumptions
+
+# Constants
+  {{$NODE_MODE := DefaultParam .NODE_MODE "allnodes"}}
+  {{$DEFAULT_NODES_PER_NAMESPACE := DefaultParam .NODES_PER_NAMESPACE 100}}
+
+# Variables
+  {{$PODS_PER_NODE := DefaultParam .PODS_PER_NODE 50}}
+  {{$PVC_PER_POD := DefaultParam .PVC_PER_POD 2}}
+  {{$ENABLE_CHAOSMONKEY := DefaultParam .ENABLE_CHAOSMONKEY false}}
+  {{$TEST_NAME := DefaultParam .TEST_NAME "generic-test"}}
+  {{$PVC_SIZE := DefaultParam .PVC_SIZE "8Gi"}}
+  {{$PER_PVC_BOUND_SLO := DefaultParam .PVC_BOUND_SLO 2}}
+  {{$POD_STARTUP_SLO := DefaultParam .POD_STARTUP_SLO 300}}
+  {{$OBJ_CREATION_QPS := DefaultParam .OBJ_CREATION_QPS 10}}
```

Comment:
what is for?

Reply:
OBJ_CREATION_QPS is a configurable variable that describes how fast to create API objects at. The default value of 10 QPS means to create it 10 queries per second. It is used to configure the UniformQPS tuning set (a thing that is responsible for creating k8s objects). 

@msau42 @wojtek-t any thoughts on what the most appropriate tuning set to test with is? I am actually not sure if sequence of QPS means in a single namespace or across all namespaces.

Reply:
so it limits how many objects in total to create in one second, which includes PVC/PV, Pods etc?

Reply:
yes thats right